nilearn/_utils/fixes/__init__.py

```diff
@@ -1,9 +1,32 @@
+# XXX use LooseVersion to specify correct breakpoint of ALL backported APIs!
+
 from distutils.version import LooseVersion
 import sklearn
 
-if (LooseVersion(sklearn.__version__) < LooseVersion('0.15') or
-      sklearn.__version__ == '0.15-git'):
-    from .sklearn_f_regression_nosparse import (
-        f_regression_nosparse as f_regression)
+if (LooseVersion(sklearn.__version__) < LooseVersion('0.14')):
+    from .sklearn_center_data import center_data
 else:
     from sklearn.feature_selection import f_regression
+
+# center_data backport
+try:
+    from sklearn.linear_models.base import center_data
+except ImportError:
+    from .sklearn_basic_backports import center_data
```

Comment:
Once we drop support for 0.12 sklearn (which is happening soon), you can remove these backports.